packages/qjackcapture/PKGBUILD

```
@@ -0,0 +1,26 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_project=QJackCapture
+pkgname="${_project,,}"
+pkgver=0.1.2
+pkgrel=1
+pkgdesc='A GUI for easy recording of JACK audio sources using jack_capture'
+arch=(any)
+url='https://github.com/SpotlightKid/qjackcapture'
+license=(GPL)
+depends=(hicolor-icon-theme jack_capture python-natsort python-pyjacklib python-pyqt5)
+makedepends=(python-setuptools)
+groups=(pro-audio)
+source=("https://files.pythonhosted.org/packages/source/q/$pkgname/$_project-$pkgver.tar.gz")
+sha256sums=('b716cee7917468a8c8589dfb3212ba09bfda43224208c96da25b8243716128bf')
+
+build() {
+  cd $_project-$pkgver
+  make
+}
+
```

There you go:
```suggestion

check() {
  cd $_project-$pkgver
  python -m qjackcapture -h > /dev/null
}

```
For me this would at lease fail `makepkg` if `python-natsort` is not installed, and passes otherwise.

This of course is nothing compared to a full test suite, but in general we should keep our own checks simple but effective if the project doesn't provide more extensive testing via `make test` or similar.